projects/angular/src/button/button-group/button-group.ts

```diff
@@ -111,6 +118,21 @@ export class ClrButtonGroup implements AfterContentInit {
     this.menuButtons = tempInMenuButtons;
   }
 
+  handleFocusOnMenuOpen() {
+    this.toggleService.openChange
+      .pipe(
+        first(isOpened => isOpened === true),
+        delay(0),
+        map(isOpened => {
+          if (isOpened && this.buttonGroupMenu) {
```

Comment:
I just realized that checking `isOpened` here is not necessary do to the `first` operator above. Also, I don't think we need to check `this.buttonGroupMenu` either. The `delay` ensures it exists. I don't think it should silently skip this code if there is a bug.

Reply:
Also, this should a `tap` operator, not a `map`. It's adding a side effect, not mapping the values.